### ISSUE TITLE
Historical Type Concepts Do Not have a "Maps To" Relationship

### ISSUE THREAD

**Issue by cukarthik** [OHDSI/Vocabulary-v5.0#614]
With the new [Type Concept](https://athena.ohdsi.org/search-terms/terms?domain=Type+Concept&standardConcept=Standard&page=1&pageSize=15&query=) domain old historical type concepts like, [Condition Type](https://athena.ohdsi.org/search-terms/terms?vocabulary=Condition+Type&page=1&pageSize=15&query=), became non-standard. However, the deprecated concepts do not have a Maps To relationship. For example, [EHR Problem List Entry](https://athena.ohdsi.org/search-terms/terms/38000245) is an orphan concept when it should most likely be mapped to [EHR Problem List.](https://athena.ohdsi.org/search-terms/terms/32840)

**Comment by cgreich** [comment]
Yeah, because neither the historical nor the new ones are external vocabularies. We try to provide mapping if you have source data with codes from a public vocabulary. These are internal. You need to refactor the ETL you have in place for the old one. Shouldn't be a big deal.

**Comment by mik-ohdsi** [comment]
@cukarthik  -  you okay with refactoring your ETL? Still think we have to implement "replaced by" links of some sort?
